exporters/jaeger/src/TUDPTransport.cc

```diff
@@ -51,7 +54,7 @@ void TUDPTransport::open()
 
   if (error)
   {
-    // TODO: log error
+    OTEL_INTERNAL_LOG_ERROR("Jaeger Exporter: getaddrinfo failed with error: " << error);
```

Comment:
This syntax looks a little weird. Could it be defined as macro with variadic parameters, then pass the current parameter as 2 parameters?

Reply:
Yeah, I agree, but probably would be more familiar for C++ developers as ostream format.It's used extensively with opencv2 (https://github.com/opencv/opencv/blob/15ba3e123f1efa5444951b638ce6bd200054820e/modules/highgui/src/backend.cpp#L78) so picked it up. 
Also, OTEL_INTERNAL_LOG_ERROR is defined as a macro with the variadic format, as it can take a single argument ( ostream) or two arguments ( ostream and k-v attribute).